### ISSUE TITLE
Please unblock our website masternodes.online

### ISSUE THREAD

**Issue by thot-mno** [MetaMask/eth-phishing-detect#4676]
Dear MetaMask team, 

we found our domain is being blocked by your extension. We have no phishing or any other ethereum related issues/code on our website and would like to ask you to remove us from your list. 

Thank you and regards
Thot for MNO

**Comment by kumavis** [comment]
plz make a pr adding it to the whitelist section of config.json

**Comment by 409H** [comment]
Duplicate. Closing in favour of https://github.com/MetaMask/eth-phishing-detect/issues/4699
